task-sdk/src/airflow/sdk/api/client.py

```diff
@@ -493,7 +493,7 @@ def noop_handler(request: httpx.Request) -> httpx.Response:
 # Note: Given defaults make attempts after 1, 3, 7, 15, 31seconds, 1:03, 2:07, 3:37 and fails after 5:07min
```

Comment:
If you adjust the count of retries, please also adjust the comments.

If you lower it to 3 then  after 15 seconds of connectivity problems a task is killed. This is happening often in my view what I see in production.

Reply:
Updated the comment